packages/frontend/src/components/CreateSwap/CreateSwap.tsx

```diff
@@ -152,6 +155,17 @@ const CreateSwap = () => {
   const spendableBalance = useSpendableBalance({ token: fromToken });
   const depositMax = isConnected ? spendableBalance : undefined;
 
+  const getTokenWithNetwork = (token: Token | undefined, chain: Chain) =>
```

Comment:
Can we put this function somewhere else? Such as in `/utils` or outside of the Component?